modules/core/src/main/java/org/apache/ignite/internal/IgnitionEx.java

```diff
@@ -289,15 +298,20 @@ public static IgniteState state(@Nullable String name) {
      * Stops default grid. This method is identical to {@code G.stop(null, cancel)} apply.
      * Note that method does not wait for all tasks to be completed.
      *
-     * @param cancel If {@code true} then all jobs currently executing on
-     *      default grid will be cancelled by calling {@link ComputeJob#cancel()}
-     *      method. Note that just like with {@link Thread#interrupt()}, it is
-     *      up to the actual job to exit from execution
+     * @param cancel If {@code true} then all jobs currently will be cancelled
+     *      by calling {@link ComputeJob#cancel()} method. Note that just like with
+     *      {@link Thread#interrupt()}, it is up to the actual job to exit from
+     *      execution. If {@code false}, then jobs currently running will not be
+     *      canceled. In either case, grid node will wait for completion of all
+     *      jobs running on it before stopping. If {@code null} then
+     *      not-{@link IgniteConfiguration#isGracefulShutdown()} is used instead.
+     * @param waitForBackups If {@code true} then node will wait until all of its data is backed up before shutting
+     *      down. If {@code null} then {@link IgniteConfiguration#isWaitForBackupsOnShutdown()} is used instead.
```

Comment:
`IgniteConfiguration#isWaitForBackupsOnShutdown()` there's no such method.

Reply:
This is public API which is planned for phase II, I will try to disable these comments.